.github/workflows/update-make-docs.yml

```diff
@@ -12,16 +12,21 @@ jobs:
 
       - name: Update procedure
         run: |
+          BRANCH=update-make-docs
+          git checkout -B "${BRANCH}"
           curl -s -Lo docs/docs.mk https://raw.githubusercontent.com/grafana/writers-toolkit/main/docs/docs.mk
           curl -s -Lo docs/make-docs https://raw.githubusercontent.com/grafana/writers-toolkit/main/docs/make-docs
           if git diff --exit-code; then exit 0; fi
-          BRANCH="$(date +%Y-%m-%d)/update-make-docs"
-          git checkout -b "${BRANCH}"
           git add .
           git config --local user.email "bot@grafana.com"
           git config --local user.name "grafanabot"
-          git commit -m "Update \`make docs\` procedure"
-          git push -v origin "refs/heads/${BRANCH}"
-          gh pr create --fill --label no-changelog --label --no-backport --label type/docs
+          git commit --allow-empty --message "Update \`make docs\` procedure"
```

Comment:
Do you want to create a PR even if there is no change? Is this for monitoring purposes so that you learn if the workflow is doing anything at all?

Reply:
I _think_ this is actually won't happen thanks to the earlier `if git diff --exit-code; then exit 0; fi`.

My thinking when I made this change is that I wouldn't want any step to fail but I realize that was looking at this step in isolation of the rest of the script and now that logic is actually confusing.

I think I might take another pass at these changes and try and more clearly express the desired logic.